config/provision/config/deploy/production.rb

```diff
@@ -0,0 +1,71 @@
+ip = '104.236.117.113'
+
+role :app, ["deploy@#{ip}"]
+role :web, ["deploy@#{ip}"]
+role :db,  ["deploy@#{ip}"]
+
+server ip, user: 'deploy', roles: %w{web app db}
+
+set :stage, 'production'
+set :rails_env, 'production'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.